maps/src/main/java/com/gluonhq/maps/MapView.java

```diff
@@ -61,12 +68,19 @@
     private boolean zooming = false;
     private boolean enableDragging = false;
     
+    private Label label;
+    
     /**
      * Create a MapView component.
      */
-    public MapView() {
-        baseMap = new BaseMap();
+    public MapView(TileRetriever tileRetriever) {
```

Comment:
For the sake of compatibility, we could keep the no-args constructor, and default to a possible implementation of `CachedOsmTileRetriever` (see comment below)?

Reply:
Thinking about it... I would prefer to not have a no-args constructor actually, for the reasons in comment above.